src/contributors/Raman.md.txt

```diff
@@ -0,0 +1,8 @@
+---
+name: "Raman Khadgi"
+username: "XxPlayerxX"
+image: "raman.jpg"
+facebook: "<your facebook username>"
+instagram: "<your instagram username>"
+twitter: "<your twitter username>"
```

Comment:
Hello @XxPlayerxX 
If you don't want to provide your social media usernames then remove them completely rather than having the placeholders only.